GeosCore/fast_jx_mod.F90

```diff
@@ -5311,6 +5319,7 @@ SUBROUTINE PHOTRATE_ADJ( Input_Opt, State_Diag, State_Met,                 &
 !
     REAL(fp) :: C_O2,     C_N2, C_H2,   ITEMPK, RO1DplH2O
     REAL(fp) :: RO1DplH2, RO1D, NUMDEN, TEMP,   C_H2O
+    REAL(fp) :: C_NIT,     C_NITs,    C_SALA,  C_SALC,  FAC
```

Comment:
Would recommend making these pointers rather than scalars, as that would repeated copy operations.

Reply:
Actually I've left the C_NIT and C_SALA as scalars. There were a lot of unused variables so I removed them.